### ISSUE TITLE
Question: Where to find myapp.yaml

### ISSUE THREAD

**Issue by mr-j0nes** [columbia-it/django-jsonapi-training#7]
Hello there,

I have taken a look at http://www.columbia.edu/~alan/schemas/common/jsonapi.yaml
it looks like a great work. I wonder if I can use it in my project?

Additionally I have found also the following link: https://columbia-it-django-jsonapi-training.readthedocs.io/en/v0.2.6/documenting-api.html
and I am trying to find the file myapp.yaml to see how to use the above specs. Any hints on where to find it ?

Thanks a lot in advance..

**Comment by n2ygk** [comment]
See https://columbia-it-django-jsonapi-training.readthedocs.io/en/latest/ (which is v0.2.7) which has been updated. myapp.yaml no longer exists and is replaced by jsonapi.yaml.

The [license](https://github.com/columbia-it/django-jsonapi-training/blob/master/LICENSE) says how you can use the code.

I'm not sure you actually want to use jsonapi.yaml but rather https://github.com/django-json-api/django-rest-framework-json-api.

**Comment by mr-j0nes** [comment]
Hello there again. Thanks for the reply and sorry for my late comeback.

Actually I am not using jdango. I am building my APIs in C++. 
For drafting my API specs I am using OAS3 and Json:PI
So I found your jsonapi.yaml which could help me draft my API specs easier.

Now, I see in your newer project (the one you linked above) that you have removed the jsonapi.yaml usage altogether. Have you found any problem with that approach?

Thanks again and wish you a happy new year..

**Comment by n2ygk** [comment]
Take a look at https://github.com/columbia-it/django-jsonapi-training/blob/master/docs/schemas/openapi.yaml#L4394-L4589 where you'll find the jsonapi components definitions.

**Comment by mr-j0nes** [comment]
Hello Alan,

thank you for pointing me in the right direction. I have already adapted my specs to the components definition in the file above.
I took the liberty of looking at the rest of the file too. Now I have a couple of questions/remarks:

1. In the file above there is an error in line 4893: two dashes instead of one (- - auth-columbia).
2. In the parameter section I see two sorts, one defined by reference (#/components/parameters/sort) and the other by defining it: name: sort. Is this intentional?
3. In /path/{id} I guess it does not make sense to accept parameter sort or filter..
4. In the relationships path you do not define the parameter page[size] and page[number]
5. I think some people in the Json:API community might be using the jsonapi.yaml shared in http://www.columbia.edu/~alan/schemas/common/jsonapi.yaml. Wouldn't it make sense to share there the newest version from above?

Thanks a lot for your help.
